serialization-avro/src/main/java/io/axoniq/dev/samples/serializationavro/SerializationAvroApplication.java

```diff
@@ -0,0 +1,41 @@
+package io.axoniq.dev.samples.serializationavro;
+
+import io.axoniq.dev.samples.serializationavro.api.Card;
+import io.axoniq.dev.samples.serializationavro.api.CardIssuedEvent;
+import io.axoniq.dev.samples.serializationavro.api.CardList;
+import io.axoniq.dev.samples.serializationavro.api.CardRedeemedEvent;
+import io.axoniq.dev.samples.serializationavro.api.GetAllCardsQuery;
+import io.axoniq.dev.samples.serializationavro.api.GetCardByIdQuery;
+import io.axoniq.dev.samples.serializationavro.api.IssueCardCommand;
+import io.axoniq.dev.samples.serializationavro.api.RedeemCardCommand;
+import org.apache.avro.message.SchemaStore;
+import org.axonframework.spring.serialization.avro.AvroSchemaScan;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Profile;
+
+@SpringBootApplication
+@AvroSchemaScan
+public class SerializationAvroApplication {
+
+    public static void main(String[] args) {
+        System.setProperty("disable-axoniq-console-message", "true");
+        SpringApplication.run(SerializationAvroApplication.class, args);
+    }
+
+    @Profile("manualSchemaStore")

```

Comment:
Wouldn't this profile clash with the `@AvroSchemaScan`? Or would the filled `SchemaStore` constructed by that annotation simply be ignored because you have this bean in place?

Reply:
So the answer is: no, it wouldn't clash. And no, there will be no pre-filling if you define your own, because the condition on that bean factory prevents that.

Reply:
Alright, thanks for elaborating!